exercises/ex1.cpp

```diff
@@ -7,42 +7,59 @@ int main()
 {
   int week;
 
-  /* Input week number from user */
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
+  switch (week)
   {
+  case 1:
+
     cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
+  
+  break;
+  
+  case 2: 
+
     cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
+
+  break;
+
+  case 3: 
+  
     cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
+  
+  break;
+
+
+  case 4: 
+
     cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
+
+  break;
+
+  case 5: 
+
     cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
+
+  break;
+
+  case 6: 
+  
     cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
+
+  break;
+
+  case 7: 
+
     cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+  
+  break;
 
+  default:
+  
+    cout << "Invalid input! Please enter week number between 1-7." << endl;  
+    break;
+  }
+  
   return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```